bin/radical-pilot-stats

```diff
@@ -990,68 +511,63 @@ def handle_doc (collection, mode, doc) :
                 lnum    += 1
 
-            print(" | | | +-- %-10s : %s" % (key, txt_out[:-1])) # remove last \n
+            print(" | | | +-- %-10s : %s" % (key, txt_out[:-1]))  # remove last \n
+
+
 # ------------------------------------------------------------------------------
 #
 def parse_commandline():
 
-    return options
+    return args
 
 
 # ------------------------------------------------------------------------------
 #
 if __name__ == '__main__' :
 
-    import optparse
-    parser = optparse.OptionParser (add_help_option=False)
+    import argparse
+    parser = argparse.ArgumentParser(description='')
 
-    parser.add_option('-s', '--session',   dest='session')
-    parser.add_option('-d', '--dburl',     dest='url')
-    parser.add_option('-m', '--mode',      dest='mode')
-    parser.add_option('-c', '--cachedir',  dest='cachedir')
-    parser.add_option('-p', '--profdir',   dest='profdir')
-    parser.add_option('-f', '--filter',    dest='filters')
-    parser.add_option('-t', '--terminal',  dest='term')
-    parser.add_option('-h', '--help',      dest='help', action="store_true")
+    parser.add_argument('-s', '--session',   dest='session')
+    parser.add_argument('-d', '--dburl',     dest='url')
+    parser.add_argument('-m', '--mode',      dest='mode')
+    parser.add_argument('-c', '--cachedir',  dest='cachedir')
+    parser.add_argument('-p', '--profdir',   dest='profdir')
+    parser.add_argument('-f', '--filter',    dest='filters')
+    parser.add_argument('-t', '--terminal',  dest='term')
 
-    options, args = parser.parse_args ()
-
-    if  args :
-        usage ("Too many arguments (%s)" % args)
-
-    if  options.help :
-        usage ()
+    args = parser.parse_args()
 
-    if  options.mode in ['help'] :
+    if  args.mode in ['help'] :
         usage ()
 
-    if  not options.mode :
+    if  not args.mode :
         usage ("No mode specified")
 
-    if  options.url :
-        default_dburl = options.url
+    if  args.url :
+        default_dburl = args.url
     else :
         if  'RADICAL_PILOT_DBURL' in os.environ :
             default_dburl = os.environ['RADICAL_PILOT_DBURL']
         else :
-            default_dburl = _DEFAULT_DBURL
+            default_dburl = _DBURL
 
-    if  not options.url :
-        options.url = default_dburl
+    if  not args.url :
+        args.url = default_dburl
 
 
-    mode     = options.mode
-    url      = options.url
-    session  = options.session
-    filters  = options.filters
-    term     = options.term
-    cachedir = options.cachedir
-    profdir  = options.profdir
+    mode     = args.mode
+    url      = args.url
+    session  = args.session
+    filters  = args.filters
+    term     = args.term
+    cachedir = args.cachedir
+    profdir  = args.profdir
 
     if  not term :
         term = "pdf,png"
 
-    url = ru.Url (options.url)
+    url = ru.Url (args.url)
     if  not url.path or '/' == url.path :
         url.path = 'radicalpilot'
```

Comment:
we shouldn't edit url, since it is provided by user (or set as env variable)

Reply:
removed.